_articles/what-is-oauth-understanding-the-authorization-layer.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: article
+title: "What is OAuth? Understanding the authorization layer"
+abstract: "A simple explanation of what OAuth is, how it works and what it is used for."
+sitemap:
+  lastmod: 2020-05-15
+---
+
+OAuth is short for Open Authorization.  OAuth is a framework for token-based authorization on the internet. What does that mean?  Basically, OAuth is a way for websites to share private information with each other. Authorization tokens are issued which grants access to specific pieces of information hence the term token-based authorization.
```

Comment:
internet -> web
This needs to talk about `OAuth 2.0` you can say from this point when we say OAuth we will be talking about OAuth 2.0.
" OAuth is a way for websites to share private information with each other" This is missing the fact it's a customers data, that is help by a website that is shared with another website.

I think some comment like the followin.
"OAuth 2.0 related but not compatible with OAuth 1.0, when talking about OAuth most people mean 2.0 but of course not all, and this is where much of the complication abount understanding OAuth comes from. 

Not can you always put a new line after each full stop. then comments can be more targeted.